package.json

```diff
@@ -16,32 +16,34 @@
     "cucumber:unused:step": "npx ./dist/src/test/e2e/config/CucumberStepManager.js",
     "report:generate": "npx allure generate --clean ./test-report/allure-result/ -o ./test-report/allure-report",
     "report:open": "npx allure open test-report/allure-report",
-    "tests:e2e": "npx del test-report && wdio ./dist/wdio.conf.js"
+    "tests:e2e": "npx del test-report && wdio ./wdio.conf.ts"
   },
   "config": {
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
   },
   "dependencies": {
-    "@wdio/allure-reporter": "^6.11.0",
-    "@wdio/cli": "^6.12.0",
-    "@wdio/cucumber-framework": "^6.11.1",
-    "@wdio/local-runner": "^6.12.0",
-    "@wdio/spec-reporter": "^6.11.0",
-    "@wdio/sync": "^6.11.0",
+    "@wdio/allure-reporter": "^7.5.2",
+    "@wdio/cli": "^7.5.2",
+    "@wdio/codemod": "^0.6.0",
+    "@wdio/cucumber-framework": "^7.5.2",
+    "@wdio/local-runner": "^7.5.2",
+    "@wdio/spec-reporter": "^7.5.2",
+    "@wdio/sync": "^7.5.2",
     "allure-commandline": "^2.13.8",
-    "chromedriver": "^87.0.5",
+    "chromedriver": "^90.0.0",
     "copyfiles": "^2.4.1",
     "cucumber": "^6.0.5",
     "del-cli": "^3.0.1",
     "expect": "^26.6.2",
     "expect-webdriverio": "^1.4.1",
     "fibers": "^5.0.0",
+    "jscodeshift": "^0.12.0",
```

Comment:
Is it necessary for this dependency (and `@wdio/codemod`) to remain after the migration is complete?
That isn't how I'd expect a codemod to work (admittedly I don't have much experience there), but from this PR and the [documentation here](https://webdriver.io/docs/v7-migration/) it seems like its intended to stick around